config/locales/en.yml

```
@@ -35,6 +35,11 @@ en:
     token_revoked: 'If your token was valid, it has been revoked'
   keywords:
     could_not_query: 'An error occurs when performing the Google Search, please try again.'
+    empty_list: 'You do not have any keyword yet.'
+  pagy:
+    errors:
+      overflow: 'There are not that many pages here! Max page is '
```

In Rails you can pass the value to the translation file
```suggestion
      overflow: "There are not that many pages here! Max page is %{max_page}"
```

Then you can call it 
```ruby
I18n.t('pagy.errors.overflow', max_page: e.pagy.last)
```

 instead of interpolating 
 ```ruby
"#{I18n.t('pagy.errors.overflow')} #{e.pagy.last}"
```

---

Very useful! :) Thanks!